homeassistant/components/fritzbox/coordinator.py

```diff
@@ -21,6 +21,13 @@
 type FritzboxConfigEntry = ConfigEntry[FritzboxDataUpdateCoordinator]
 
 
+def pre_load_light_capabilities(device: FritzhomeDevice) -> None:
+    """Pre-load light capabilities for lightbulbs."""
+    if device.has_lightbulb and device.has_color:
+        device.supported_colors = device.get_colors()
+        device.supported_color_temps = device.get_color_temps()
```

Comment:
So wait, we are assigning the fields to a non-known field of `device`? That feels dirty

Reply:
I agree - but I couldn't find another way to do this without changes to the backend library.
Maybe the code owners can chip in with thoughts?

Reply:
I've reverted the pre-load, and this now matches the previous behavior

Reply:
we can preload the supported colors and color temps within the coordinator and save them in an new attribute in `FritzboxCoordinatorData` - will prepare a poc for this

Reply:
- #133798